database_sanitizer/tests/test_sanitizers_user.py

```diff
@@ -59,9 +59,9 @@ def test_sanitize_surname_en_gb():
 
 
 def test_sanitize_email_resets_on_session_reset():
-    assert user.sanitize_email('test@example.com') == (
-        'zoe.burke@xce13103b.sanitized.net')
+    assert user.sanitize_email('test@example.net') == (
```

Comment:
You can't change the source address and expect it to sanitize to same value. :)

Reply:
Ah.. we had the random hash there. 👍 Will check what it gave and update the expected value. I think having different source domain helps to understand the functionality easier.